<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLMarkupsSphereWidget</class>
 <widget class="QWidget" name="qMRMLMarkupsSphereWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="sphereCollapsibleButton">
     <property name="text">
      <string>Sphere Settings</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="radiusModeLabel">
          <property name="text">
           <string>Radius
mode:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="radiusModeComboBox">
          <property name="toolTip">
           <string>Centered : first control point is the centre of the sphere, the second one is on the circumference.

Circumferential : both control points lie on the circumference.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="resolutionLabel">
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ctkSliderWidget" name="resolutionSliderWidget">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="singleStep">
           <double>6.000000000000000</double>
          </property>
          <property name="pageStep">
           <double>24.000000000000000</double>
          </property>
          <property name="minimum">
           <double>12.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>36.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="drawModeLabel">
          <property name="text">
           <string>Slice draw
mode:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="drawModeComboBox">
          <property name="toolTip">
           <string>World intersection : the slice plane cuts the sphere in 3D view, and the outline is drawn in the slice view.

Slice projection : the slice plane cuts a virtual sphere, created from the projection of the control points in slice views, and the outline is drawn.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLWidget" name="resliceMRMLWidget">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="reslicePushButton">
           <property name="toolTip">
            <string>Reslice the selected node to be co-planar to the control points.</string>
           </property>
           <property name="text">
            <string>Reslice:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="resliceInputSelector">
           <property name="toolTip">
            <string>Make this node co-planer to the currently drawn radius.</string>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLSliceNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>true</bool>
           </property>
           <property name="addEnabled">
            <bool>false</bool>
           </property>
           <property name="removeEnabled">
            <bool>false</bool>
           </property>
           <property name="selectNodeUponCreation">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>resliceMRMLWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>resliceInputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
